src/AentHelper.php

```diff
@@ -138,4 +138,51 @@ public function question(string $question): \TheAentMachine\Helper\Question
     {
         return new \TheAentMachine\Helper\Question($this->questionHelper, $this->input, $this->output, $question);
     }
+
+    /**
+     * @param string[] $choices
+     * @return Helper\ChoiceQuestion
+     */
+    public function choiceQuestion(string $question, array $choices): \TheAentMachine\Helper\ChoiceQuestion
+    {
+        return new \TheAentMachine\Helper\ChoiceQuestion($this->questionHelper, $this->input, $this->output, $question, $choices);
+    }
+
+    public function setEnvType(): string
+    {
+        $envType = $this->choiceQuestion('Select your environment type', ['DEV', 'TEST', 'PROD'])
```

Comment:
We should replace those with some constants (will be easier to retrieve metadata).

Reply:
Oki doki !